doc/mgr/ansible.rst

```diff
@@ -120,3 +120,56 @@ Example::
     sda (hdd, 53687091200b)
     sdb (hdd, 53687091200b)
     sdc (hdd, 53687091200b)
+
+**Create OSD:**
+
+Create an OSD in one or more hosts using a drive group specification:
+
+orchestrator osd create {<osd_host>} {<json>} {<file>}
+
+Where:
+
+{<osd_host>}: Optional parameter to create OSD only in this host
+{<json>}: String with json format used to specify 'lvm_volumes' parameter
+{<file>}: Name of the file that contains a json string to specify 'lvm_volumes' parameter
+
+The "lvm_volumes" parameter is a list of dictionaries that contains the Drive group attributes
```

Comment:
where does `lvm_volumes` come from`

Reply:
"lvm_volumes" is the parameter used in Ceph_ansible to group parameters for configure the composition of the OSD ( what drives are used for data, journal/db, wal)
With the last modification , we start to use an "agnostic" and "abstract"  "drive group" specification. So the redaction of this paragraph changes substantially.